.release/ci.hcl

```diff
@@ -0,0 +1,250 @@
+schema = "1"
+
+project "envconsul" {
+  // the team key is not used by CRT currently
+  team = "cat-floss"
+  slack {
+    notification_channel = "C026W707YHJ"
+  }
+  github {
+    organization = "hashicorp"
+    repository = "envconsul"
+    release_branches = ["master"]
```

Comment:
👀  Can we update this to `main`? :D

Reply:
Yep. I converted all my repos to main just a couple weeks ago. I think I missed this in all of them. :p  I'll go fix that.

Reply:
Corrected everywhere. Thanks!